src/OSPSuite.Core/Domain/Services/DosingInterval.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace OSPSuite.Core.Domain.Services
+{
+   public class DosingInterval
+   {
+      /// <summary>
+      ///    Time value representing the beginning of the interval in [min]
+      /// </summary>
+      public float? StartValue { get; set; }
+
+      /// <summary>
+      ///    Time value representing the end of the dosing interval in [min]
+      /// </summary>
+      public float? EndValue { get; set; }
+
+      /// <summary>
+      ///    Dose (in µmol/kg BW) of the interval
+      /// </summary>
+      public double? Dose { get; set; }
```

Comment:
Not sure why it's in µmol/kg BW here; but then I would rename it to **DrugMassPerBodyWeight**
(we have usually Dose in mass units and DrugMass in molar units)